8Knot/pages/affiliation/visualizations/gh_org_affiliation.py

```diff
@@ -23,10 +23,28 @@
     [
         dbc.CardBody(
             [
-                html.H3(
-                    "Organization Affiliation by GitHub Account Info",
-                    className="card-title",
-                    style={"textAlign": "center"},
+                dbc.Row(
+                    [
+                        dbc.Col(
+                            html.H3(
+                                "Organization Affiliation by GitHub Account Info",
+                                className="card-title",
+                            ),
+                        ),
+                        dbc.Col(
+                            dbc.Button(
+                                "About Graph",
+                                id=f"popover-target-{PAGE}-{VIZ_ID}",
+                                color="outline-secondary",
+                                size="sm",
+                                className="about-graph-button",
+                            ),
+                            width="auto",
+                        ),
+                    ],
+                    align="center",
+                    justify="between",
+                    className="mb-3",
```

Comment:
it seems like we are making the same changes here across several visualizations.

Would it be better (maybe not now but in future) to have the shared elements defined centrally (in a function or superclass) so that they can just be imported by each visualization so the the specific values for the titles and about information can be passed in while keeping styling somewhat centralized (yet overrideable for special cases)